pkg/model/components/context.go

```diff
@@ -125,8 +126,16 @@ func Image(component string, clusterSpec *kops.ClusterSpec) (string, error) {
 		return "gcr.io/google_containers/kubedns-amd64:1.3", nil
```

Comment:
We may actually need to remove this from the API.  This is not being used at all and really need to get out of the API.  It is an Addon ...